### PR TITLE
Feature/user service response codes

### PR DESCRIPTION
**Pull request by lrschaef**
Reasons for this PR:
- Add response code defined in API

Changes in this PR:
- Added response codes
- Extended some model classes with functionality for error-checking

### CODE REVIEW

File: product_code/lagom/user_service/api/src/main/scala/de/upb/cs/uc4/user/model/Address.scala [hunk 1]
```diff
@@ -11,6 +11,15 @@ case class Address(street: String,
   def trim: Address = {
     copy(street.trim, houseNumber.trim, zipCode.trim, city.trim, country.trim)
   }
+
+  def oneEmpty: Boolean = {
+    if(street == "" || houseNumber == "" || zipCode == "" || city == "" || country == "") {
+      true
+    }
+    else {
```
Comment (ldklenner) [inline product_code/lagom/user_service/api/src/main/scala/de/upb/cs/uc4/user/model/Address.scala:19]: Reconsider the formatting of this line

Comment (lrschaef) [reply product_code/lagom/user_service/api/src/main/scala/de/upb/cs/uc4/user/model/Address.scala:19]: Can you give an idea, how you would format this, or what exactly should be reconsidered? Is it the position of the else?

Comment (david-buderus) [reply product_code/lagom/user_service/api/src/main/scala/de/upb/cs/uc4/user/model/Address.scala:19]: just write is as one line
```scala 
def oneEmpty: Boolean = {street == "" || houseNumber == "" || zipCode == "" || city == "" || country == ""}
```

Comment (ldklenner) [reply product_code/lagom/user_service/api/src/main/scala/de/upb/cs/uc4/user/model/Address.scala:19]: Yes, it is the position of the Else :smile: Is this a personal preference or recommented by the scala style guide?

Comment (lrschaef) [reply product_code/lagom/user_service/api/src/main/scala/de/upb/cs/uc4/user/model/Address.scala]: Fixed, closing conversation.

**Pull request merged**